dsf-fhir/dsf-fhir-server/src/main/java/org/highmed/dsf/fhir/authorization/ResearchStudyAuthorizationRule.java

```diff
@@ -145,29 +145,29 @@ public ResearchStudyAuthorizationRule(DaoProvider daoProvider, String serverBase
 			errors.add("ResearchStudy.enrollment one or more Group references missing");
 		}
 
-		if (newResource.hasPrincipalInvestigator())
-		{
-			Optional<Resource> practitioner = resolvePractitioner(connection, user,
-					"ResearchStudy.principalInvestigator", newResource.getPrincipalInvestigator());
-			if (practitioner.isPresent() && practitioner.get() instanceof Practitioner
-					&& ((Practitioner) practitioner.get()).getActive())
-			{
-				if (!practitionerRoleExists(connection, user, practitioner.get().getIdElement()))
-				{
-					errors.add(
-							"ResearchStudy.principalInvestigator corresponding PractitionerRole.practitioner not found");
-				}
-			}
-			else
-			{
-				errors.add(
-						"ResearchStudy.principalInvestigator not resolved or not instance of Practitioner or not active");
-			}
-		}
-		else
-		{
-			errors.add("ResearchStudy.principalInvestigator missing");
-		}
+//		if (newResource.hasPrincipalInvestigator())
+//		{
+//			Optional<Resource> practitioner = resolvePractitioner(connection, user,
+//					"ResearchStudy.principalInvestigator", newResource.getPrincipalInvestigator());
+//			if (practitioner.isPresent() && practitioner.get() instanceof Practitioner
+//					&& ((Practitioner) practitioner.get()).getActive())
+//			{
+//				if (!practitionerRoleExists(connection, user, practitioner.get().getIdElement()))
+//				{
+//					errors.add(
+//							"ResearchStudy.principalInvestigator corresponding PractitionerRole.practitioner not found");
+//				}
+//			}
+//			else
+//			{
+//				errors.add(
+//						"ResearchStudy.principalInvestigator not resolved or not instance of Practitioner or not active");
+//			}
+//		}
+//		else
+//		{
+//			errors.add("ResearchStudy.principalInvestigator missing");
+//		}
```

Comment:
why was this commented out?

Reply:
Feasibility requests are designed to be anonymous, therefore no PI is needed in the research study resource. After adding the data sharing process, this rule needs to be based on the profile of the research study, since data sharing research studies need a PI

Reply:
To make the PI optional, leave 148-166 and only remove 167-170

Reply:
Ah I see, if there is a PI in the study, it should be validated.